.azure-pipelines/linux.yml

```diff
@@ -53,3 +53,34 @@ jobs:
       pathtoPublish: "$(Build.StagingDirectory)/envoy"
       artifactName: $(CI_TARGET)
     condition: always()
+
+- job: fuzzit
```

Comment:
Now this job can be rolled in to the parameterized job above?

Reply:
Oh yes, that would make sense for the regression but how do I make the fuzzing only run on pushes to master / daily cron (by fuzzing I mean building the fuzzers and uploading them to fuzzit so they can run asynchronously and I want this to happen only for the master branch and not every PR /branch etc...)

Reply:
ah, then probably not rolling into is fine. You can add a conditions field to the job, `condition: ne(variables['Build.Reason'], 'PullRequest')`, to exclude PRs? See https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml for details and tune it to what you want to do.

Reply:
ok I'll try that. I'll try to roll regression into that parametrised and the fuzzing into it's own step with `condition`

Reply:
that's fine too, thanks!